### ISSUE TITLE
ESP32 "Guru Meditation Error" after inserting "#include <WiFiManager.h>" into sketch

### ISSUE THREAD

**Issue by jkrasinger** [tzapu/WiFiManager#1399]
#### Hardware
WiFimanager Branch/Release:  2.0.11-beta

Esp32: 

Hardware:			ESP32 DEV KIT V1

Core Version:      	ESP32 2.0.2

### Description

After adding "#include <WiFiManager.h>" to sketch without any other modifications to the Program get 
"Guru Meditation Error: Core  1 panic'ed (LoadProhibited). Exception was unhandled."

This error seems to happen in the NTPClient routines.

"

### Settings in IDE

Module:  			DOIT ESP32 DEVKIT V1

Additional libraries:  TFT, TFT_eSPI, NTPClient, PubSubClient, WiFiUdp

### Debug Messages

```
messages here
.....Attempting MQTT connection...MQTT connected
Guru Meditation Error: Core  1 panic'ed (LoadProhibited). Exception was unhandled.

Core  1 register dump:
PC      : 0x400d7177  PS      : 0x00060430  A0      : 0x800d8575  A1      : 0x3ffb2750  
A2      : 0x00000000  A3      : 0x00000000  A4      : 0x00000000  A5      : 0x00000000  
A6      : 0x00000000  A7      : 0x00000000  A8      : 0x800de964  A9      : 0x3ffb2760  
A10     : 0x3ffb27cc  A11     : 0x00000000  A12     : 0x00000001  A13     : 0x3ffc337c  
A14     : 0x3ffc39a4  A15     : 0x3ffb2774  SAR     : 0x0000000a  EXCCAUSE: 0x0000001c  
EXCVADDR: 0x00000000  LBEG    : 0x4008974d  LEND    : 0x4008975d  LCOUNT  : 0xffffffff  


Backtrace:0x400d7174:0x3ffb27500x400d8572:0x3ffb2790 0x400d32af:0x3ffb27c0 0x400d365a:0x3ffb2800 0x400df03d:0x3ffb2820

**Comment by tablatronix** [comment]
I need a sketch

**Comment by jkrasinger** [comment]
I understand, but the sketch is a big one, should i really add it to the bug report?

**Comment by tablatronix** [comment]
No you should use a miminal reproduction sketch, like an example from the library

**Comment by jkrasinger** [comment]
Excuse me for the late response, but i tried to reproduce the error with a minimalistic sketch, but i was not able to reproduce it. I first assumed the problem within the WiFi routines, but this was not the case, they worked as expected. The next component to check was the TFT-Touch screen with the libraries TFT, TFT_eSPI in use. I deactivated the code step by step until i reached the "check for a touch" on the screen. As soon as i disabled this routine all was working as expected. So i made a deeper dive in this routine ... and found the bug in my code. I made a loop over 15(???) touch buttons instead of three. I have no idea why this happened. After correcting this little mistake the sketch was up and running again.
This sketch was working even with this error, but by only inserting the "#include <WiFiManager.h>" without using the WiFiManager(!) my sketch died. This is very interesting.

Excuse me for the false alarm, i will close this issue shortly.
Thank you for your quick response.
